quickstarts/microsoft.web/webapp-custom-deployment-slots/azuredeploy.json

```diff
@@ -62,7 +70,19 @@
         "displayName": "WebApp"
       },
       "properties": {
-        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanName'))]"
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanName'))]",
+        "siteConfig": {
+          "appSettings": [
+            {
+              "name": "[parameters('appSettingName')]",
+              "value": "GEN-UNIQUE"
```

Comment:
GEN tokens are not valid in a template file, only param files - what should these value be?